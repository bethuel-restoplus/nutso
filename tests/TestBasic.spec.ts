import { nutso } from "../index";

test(`Basic`, () => {
  expect(nutso()).toMatchSnapshot();
});
