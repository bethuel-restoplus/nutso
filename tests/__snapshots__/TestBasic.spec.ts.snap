// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic 1`] = `undefined`;
